src/org/opensolaris/opengrok/history/FileHistoryCache.java

```diff
@@ -64,10 +64,11 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(FileHistoryCache.class);
 
     private final Object lock = new Object();
-    private String historyCacheDirName = "historycache";
-    private String latestRevFileName = "OpenGroklatestRev";
+    private final String historyCacheDirName = "historycache";
+    private final String latestRevFileName = "OpenGroklatestRev";
     private boolean historyIndexDone = false;
-    
+    public final static String historyDirName = "historycache";
```

Comment:
Yes the test can live without it. Or if you make the property protected it should be accessible from the test as it is the same package.

Doesn't NB hint you with naming conventions for constants?